api/src/pcapi/routes/backoffice/templates/admin/roles.html

```diff
@@ -1,32 +1,50 @@
+{% from "components/turbo/spinner.html" import build_loading_spinner with context %}
+{% import "components/presentation/details/tabs.html" as tabs %}
 {% extends "layouts/admin.html" %}
 {% block main_content %}
   <div class="container">
     <h2>Gestion des rôles</h2>
-    {% for form in forms %}
-      <div class="card mb-4">
-        <div class="card-header"
-             data-bs-toggle="collapse"
-             href=".pc-collapse-{{ form.name }}"
-             role="button">{{ form.name }}</div>
-        <div class="collapse card-collapse p-3 shadow pc-collapse-{{ form.name }}">
-          <form class=".{{ form.name }}"
-                action="{{ url_for(".update_role", role_id=form.id) }}"
-                method="post"
-                name="{{ url_for(".update_role", role_id=form.id) | action_to_name }}">
-            <div class="card-body">
-              <div class="row">
-                {% for form_field in forms[form] %}{{ form_field }}{% endfor %}
-              </div>
-              <div class="row">
-                <div class="my-3">
-                  <button type="submit"
-                          class="btn btn-sm py-2 btn-primary">Sauvegarder les modifications</button>
-                </div>
+    <div class="mt-4">
+      {% call tabs.build_details_tabs_wrapper() %}
+        {{ tabs.build_details_tab("gestion", "Gestion", True) }}
+        {{ tabs.build_details_tab("historique", "Historique") }}
```

Comment:
On met en général les noms "techniques" en anglais, seul l'affichage est en français.
Donc par cohérence avec le reste du BO : 
```suggestion
        {{ tabs.build_details_tab("management", "Gestion", True) }}
        {{ tabs.build_details_tab("history", "Historique") }}
```
et pareil dans les `build_details_tab_content` en-dessus